ONSdigital/ras-party
# updated urllib3 version

# Motivation and Context
The Travis builds were failing due to a lint error. The cause was urllib3 not being updated to the latest version.

# What has changed
- Upgraded the Pipfile and Pipfile.lock to use version 1.25.8 of urllib3.

# How to test?
- Run `make lint` (may need to run `make build` beforehand) and make sure there aren't any lint errors

# Links
[Trello card](https://trello.com/c/L9GaddQJ)
